Reword and correct punctuation on installation.rst (#643)
nipy/heudiconv: docs/installation.rst

=====
Released versions of HeuDiConv are available on `PyPI <https://pypi.org/project/heudiconv/>`_
and `conda <https://github.com/conda-forge/heudiconv-feedstock#installing-heudiconv>`_.

> To install through ``PyPI``, run::

Reviewer: I don't think this `,` is needed

Reviewer: damn chatgpt, it overruled me again

    pip install heudiconv[all]

> On Debian-based systems, we recommend using `NeuroDebian <http://neuro.debian.net>`_

Reviewer: chatgpt said we need it, poor mortal Yarik surrenders to the wisdom of Isaac & ChatGPT

    $ docker pull nipy/heudiconv:latest

Note that when using via ``docker run``, you might need to provide your user and group IDs so they map correspondingly
within container, i.e. like::

    $ docker run --user=$(id -u):$(id -g) -e "UID=$(id -u)" -e "GID=$(id -g)" --rm -t -v $PWD:$PWD nipy/heudiconv:latest [OPTIONS TO FOLLOW]

> Additionally, `the ReproIn heuristic project <https://reproin.repronim.org>`_ provides its own Docker images on

Reviewer: re comment by @candleindark [below](https://github.com/nipy/heudiconv/pull/643/files#r1107625218). Good point, what about minimalistic:

```suggestion
Additionally, `the ReproIn heuristic project <https://reproin.repronim.org>`_, which develops the ``reproin`` heuristic,
also provides its own Docker images on
```

> Docker Hub at `repronim/reproin <https://hub.docker.com/r/repronim/reproin>`_ which bundle its `reproin` helper.

Author: The transition to talking about the ReproIn project is rather abrupt here. It would be helpful if there is some information about the relationship between HeudiConv and ReproIn before this transition.

    $ singularity pull docker://nipy/heudiconv:latest

Singularity YODA style using ///repronim/containers
===================================================
`ReproNim <https://www.repronim.org/>`_ provides a large collection of Singularity container images of popular
neuroimaging tools, e.g. all the BIDS-Apps. This collection also includes the forementioned container
images for `HeuDiConv <https://github.com/ReproNim/containers/tree/master/images/nipy>`_ and
`ReproIn <https://github.com/ReproNim/containers/tree/master/images/repronim>`_ in the Singularity image format. This collection is available as a
`DataLad <https://datalad.org>`_ dataset at `///repronim/containers <http://datasets.datalad.org/?dir=/repronim/containers>`_

> on `Datalad.org <http://datalad.org>`_ and as `a GitHub repo <https://github.com/ReproNim/containers>`_.

Reviewer: ```suggestion
on `datasets.datalad.org <http://datasets.datalad.org>`_ and as `a GitHub repo <https://github.com/ReproNim/containers>`_.
```

The HeuDiConv and ReproIn container images are named ``nipy-heudiconv`` and ``repronim-reproin``, respectively, in this collection.
To use them, you can install the DataLad dataset and then use the ``datalad containers-run`` command to run.
For a more detailed example of using images from this collection while fulfilling
the `YODA Principles <https://github.com/myyoda/poster/blob/master/ohbm2018.pdf>`_, please check out
`A typical YODA workflow <https://github.com/ReproNim/containers#a-typical-yoda-workflow>`_ in
the documentation of this collection.

**Note:** With the ``datalad containers-run`` command, the images in this collection work on macOS (OSX)

> as well for ``datalad containers-run`` automagically takes care of running the Singularity containers via Docker.

Reviewer: ```suggestion
as well for ``repronim/containers`` helpers automagically take care of running the Singularity containers via Docker.
```

Singularity YODA style using ///repronim/containers
===================================================

The ReproNim project provides `///repronim/containers <http://datasets.datalad.org/?dir=/repronim/containers>`_
(git clone present also on `GitHub <https://github.com/ReproNim/containers>`__) `DataLad
<https://datalad.org>`_ dataset with Singularity containers for many popular neuroimaging tools, e.g. all BIDS-Apps.

> It also contains converted from Docker singularity images for stock heudiconv images (as `nipy-heudiconv

Reviewer: ```suggestion
It contains converted from Docker singularity images for stock heudiconv images (as `nipy-heudiconv
```

<https://github.com/ReproNim/containers/tree/master/images/nipy>`__) and reproin images (as `repronim-reproin

> <https://github.com/ReproNim/containers/tree/master/images/repronim>`__). Please see `"A typical workflow"

Author: The first two sentences of this paragraph are very confusing. Please reword.

> <https://github.com/ReproNim/containers/tree/master/images/repronim>`__). Please see `"A typical workflow"

Author: The first two sentences of this paragraph are very confusing. What are the intended ideas?
